src/Shell/BootstrapShell.php

```diff
@@ -66,7 +67,7 @@ public function modifyView($path = null)
      *
      * @return \Cake\Console\ConsoleOptionParser
      */
-    public function getOptionParser()
+    public function getOptionParser(): \Cake\Console\ConsoleOptionParser
```

Comment:
Add "use" statement and just use class name here, not FQCN. Similar change is required in other files too.